latest/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -235,6 +235,54 @@ enough space for the copy;  upgrade failure because of not enough space might
 lead to data loss.
 
 
+Extending image
+----------------
+
+This image can be extended using
+[source-to-image](https://github.com/openshift/source-to-image).
+
+For example to build customized image `new-postgresql`
+with configuration in `~/image-configuration/` run:
+
+
+```
+$ s2i build --assemble-user root ~/image-configuration/ postgresql new-postgresql
+```
+
+The directory passed to `s2i build` should contain one or more of the
+following directories:
+
+##### `postgresql-config/`
+
+when running `run-service` command contained
```

Comment:
I would restate a bit differently since right now it seems a bit confusing. Something along the lines of:

The `postgresql.conf` configuration file contained in this folder is used instead of the default configuration when `postgresql` is started

Reply:
This sounds bad to me. Shouldn't we again rather only "include" (additional?) configuration file into the default postgresql.conf?